mastodon/src/main/java/org/joinmastodon/android/GlobalUserPreferences.java

```diff
@@ -139,6 +140,7 @@ public static void load(){
 		replyLineAboveHeader=prefs.getBoolean("replyLineAboveHeader", true);
 		compactReblogReplyLine=prefs.getBoolean("compactReblogReplyLine", true);
 		confirmBeforeReblog=prefs.getBoolean("confirmBeforeReblog", false);
+		hapticFeedback=prefs.getBoolean("hapticFeedback", true);
```

Comment:
You should put `Build.VERSION.SDK_INT >= Build.VERSION_CODES.R` instead of the default true, so that the settings toggle stays consistent with what should happem

Reply:
Good call.